src/components/content/order/serviceActions/elements/NumberInput.tsx

```diff
@@ -0,0 +1,60 @@
+import { QuestionCircleOutlined } from '@ant-design/icons';
+import { Form, InputNumber, Tooltip } from 'antd';
+import { Rule } from 'rc-field-form/lib/interface';
+import React from 'react';
+import serviceOrderStyles from '../../../../../styles/service-order.module.css';
+import { ServiceChangeParameter } from '../../../../../xpanse-api/generated';
+import { DeployVariableSchema } from '../../types/DeployVariableSchema';
+
+export function NumberInput({ actionParameter }: { actionParameter: ServiceChangeParameter }): React.JSX.Element {
```

Comment:
these files look like a lot of copy of existing code. can we make it common such that it works with deploy variable, config parameter and as well as action parameter. 

Applies to all input types.